website/docs/cli/test/index.mdx

```diff
@@ -0,0 +1,60 @@
+- Terraform v1.6.0 deprecated the experimental Terraform `test` command and released the updated and finalized Terraform [`test`](/terraform/cli/commands/test) command.
+
+Note the introduction and deprecation of the experimental `test` command, followed by the introduction of the finalized `test` command. Read the [v1.6.x Upgrade Guide](/terraform/language/v1.6.x/upgrade-guides) for a summary of the changes between the experimental and finalized command.
+
+## The Terraform `test` command
+
+The Terraform `test` command:
+
+- Locates Terraform testing files within your configuration directory.
+- Provisions the infrastructure within your configuration as specified by each testing file.
+- Runs the assertions from the test file against the provisioned infrastructure.
+- Destroys the provisioned infrastructure at the end of the test.
+
+The `test` command, along with command-line flags and options, is discussed in detail within the [Command: test](/terraform/cli/commands/test) page.
+
+### Writing configuration for your tests
+
+Terraform test files have their own configuration syntax. This is discussed in detail within the language [Test Files](/terraform/language/tests) page.
+
+The test file syntax is focused on customizing Terraform executions for the current configuration, and overriding variables and providers to test different behaviours.
+
+## Validations
+
+Validations allow you to verify aspects of your configuration and infrastructure as it is applied and created. Terraform Cloud also supports automated [Continuous Validation](/terraform/cloud-docs/workspaces/health#continuous-validation).
+
+The Terraform `test` command will also execute all validations within your configuration as part of any tests it executes. For more information on the available validations read the [Checks](/terraform/language/checks) and [Custom Condition](/terraform/language/expressions/custom-conditions) language pages.
+
+## Tests or Validations
+
+You can write many validations as test assertions, but there are specific use cases for both.
+
+Validations are executed during usual Terraform plan and apply operations, and are also checked as part of any executed tests. Therefore, validations should be used to validate aspects of your configuration that should always be true and can impact the valid execution of your infrastructure. Module authors should also note that validations are executed and exposed to module users should they fail, and should be understandable to the user and actionable.
+
+In contrast, tests are only executed when requested and can be used to simulate specific behaviours within the configuration. Therefore, tests should be used to assert the correctness of any logical operations or behaviour within your configuration. For example, any conditional resources created based on an input can be verified as part of a test by setting the input that controls the conditional resource to true.
```

Comment:
I tried rewording this in active voice, but I'm not super happy with the last sentence yet. Would love to know what you think and we can keep iterating! 

```suggestion
In contrast, Terraform only executes tests when you run `terraform test`. Use tests to assert the correctness of any logical operations or specific behavior within your configuration. For example, you can test that Terraform creates conditional resources based on an input by setting the input controlling those resources to a certain value then verifying the resources Terraform creates. 
```

Reply:
Yeah, I think that sounds fine. I also wasn't particularly happy with my phrasing either - tests vs validations are very similar so it's a fine line to walk when deciding between them which does make providing an example here difficult.